src/extension/background-script/connectors/commando.ts

```diff
@@ -107,6 +139,32 @@ export default class Commando implements Connector {
     await this.ln.disconnect();
   }
 
+  get supportedMethods() {
+    return supportedMethods;
+  }
+
+  methodDescription(method: string) {
+    return `commando.${method}`;
```

Comment:
as we now have the description in the translation file and only use the method here I am wondering if we still need this method or if we move this convention to the `request` action. 

we can set the `description` there to ``${connector.constructor.name.toLowerCase()}.${method}`` 
then we don't need the `methodDescription` function in the connector. 

we only need to check if there is a translation for that - though as long as we have a whitelist we can make it a requirement to have a translation. 
it only will no longer work if we allow everything (and not only whitelisted ones) - though I'd for now still stick with this whitelist.

Reply:
Agree. Will refactor this part 👍